script/tool/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
   `--no-push-flags`. Releases now always tag and push.
 - **Breaking change**: `publish`'s `--package` flag has been replaced with the
   `--packages` flag used by most other packages.
+- Formatting now skips files that contain a line that exactly
+  matches the string `// This file is hand-formatted`.
```

Comment:
The period needs to be inside the \`s; it's part of the string being searched for.